tests/unit/Db/FailedLoginAttemptMapperTest.php

```diff
@@ -115,14 +115,13 @@ public function tearDown(): void {
 	}
 
 	public function testGetSuspiciousActivityCountForUidIpCombination() {
-		$functionCallTime = $this->baseTime+110;
```

Comment:
@phil-davis with this value of `$functionCallTime`  `$this->mapper->getSuspiciousActivityCountForUidIpCombination('test1', '192.168.1.1')` returns a boolean `false` which is not expected!
with `$functionCallTime = $this->baseTime+60;` then `$this->mapper->getSuspiciousActivityCountForUidIpCombination('test1', '192.168.1.1')` returns `2` which is also not expected!
Any suggesstions?